salt: Allow to provide featureGates for apiserver in bootstrap config (#3319)
scality/metalk8s: docs/installation/bootstrap.rst

        minion: <hostname-of-the-bootstrap-node>
      archives:
        - <path-to-metalk8s-iso>
      kubernetes:

>         apiserver:

Reviewer: This...

Author: Oh ! Right Typo here

>           featuresGates:

Reviewer: This should be `featureGates` (also in the implementation, if applicable).

Author: --' MB again, I did it too fast :smiley:

the bootstrap script is executed, those ISOs are automatically mounted and the
system is configured to re-mount them automatically after a reboot.

The ``kubernetes`` field can be omitted if you do not have any specific
Kubernetes `Feature Gates`_ to enable or disable.

> If you need to enable or disable a specific features for ``kube-apiserver``

Reviewer 2: ```suggestion
If you need to enable or disable specific features for ``kube-apiserver``
```

Author: Oh ! Fixed

>     If you want to enable or disable a specific feature for ``kube-apiserver``

Reviewer: Why is this indented?

Author: Just to add several paragraphs for various "kubernetes" components in the future

Author: Maybe should remove this indentation here since we will release this version with only apiserver

Author: I removed indentation here

>     If you want to enable or disable a specific feature for ``kube-apiserver``

Reviewer: ```suggestion
    If you need to enable or disable specific features of ``kube-apiserver``
```

>     just put this one in ``kubernetes.apiServer.featureGates``.

Reviewer: .. and this are not aligned (`apiserver` vs. `apiServer`).

>     just put this one in ``kubernetes.apiServer.featureGates``.

Reviewer: ```suggestion
    configure the corresponding entries in the ``kubernetes.apiServer.featureGates`` mapping.
```
